packages/block-editor/src/components/block-list/style.scss

```diff
@@ -657,6 +666,8 @@
 
 	.is-dragging-components-draggable & {
 		opacity: 0;
+		// Use a minimal duration to delay hiding the element, see hide-during-dragging animation for more details.
```

Comment:
It might be good to explain in the comment why the toolbar/popover needs to be hidden, mentioning that it obscures drag events if it remains in position overlapping post content.